templater/models_test.go

```diff
@@ -0,0 +1,28 @@
+package templater_test
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	. "github.com/opencontrol/fedramp-templater/templater"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("ControlTable", func() {
+	Describe("Fill", func() {
```

Comment:
> is there a way of getting that xml data and putting it in here as a string?

It would be _huge_, so would prefer to keep it as a separate file. Went with templatizing the file to make it a bit more flexible.